CHANGELOG.md

```diff
@@ -49,6 +56,45 @@ Lotus now offers two versions of its Ethereum-compatible APIs (`eth_*`, `trace_*
   * Removed internal `Eth*Limited` methods (e.g., `EthGetTransactionByHashLimited`) from the supported gateway API surface.
   * Improved error handling: block selection endpoints now consistently return `ErrNullRound` (and corresponding JSONRPC errors) for null tipsets.
 
+## 🐛 Bug Fix Highlights
+- fix(eth): apply limit in EthGetBlockReceiptsLimited ([filecoin-project/lotus#12883](https://github.com/filecoin-project/lotus/pull/12883))
+- fix(eth): always return nil for eth transactions not found ([filecoin-project/lotus#12999](https://github.com/filecoin-project/lotus/pull/12999))
+- fix(deps): fix Ledger hardware wallet support ([filecoin-project/lotus#13048](https://github.com/filecoin-project/lotus/pull/13048))
+
+## 📝 Changelog
+
+For the full set of changes since the last stable release:
+
+- Node: https://github.com/filecoin-project/lotus/compare/v1.32.3...v1.33.0-rc1
```

Comment:
```suggestion
- Node: https://github.com/filecoin-project/lotus/compare/release/v1.32.3...release/v1.33.0
```

The link didn't work for me otherwise

Reply:
Committing this before I make a proposal on top.